website/meta/universe.json

```diff
@@ -1967,6 +1967,28 @@
             "category": ["pipeline"],
             "tags": ["inflection", "lemmatizer"]
         },
+        {
+            "id": "amrlib",
+            "slogan": "A python library that makes AMR parsing, generation and visualization simple.",
+            "description": "amrlib is a python module and SpaCy add-in for Abstract Meaning Representation (AMR).  The system can parse sentences to AMR graphs or generate text from existing graphs.  It includes a GUI for visualization and experimentation.",
```

Comment:
Another FYI - we prefer the spelling "spaCy", i.e. without capitalizing the first letter ;-)